config/routes.rb

```diff
@@ -2,6 +2,10 @@
 
   devise_for :users
 
+  # special omniauth routes
+  get 'auth/:provider/callback', to: 'omniauth#callback'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.